src/api/app/models/comment.rb

```diff
@@ -50,18 +50,17 @@ def create_notification(params = {})
 
   # build an array of users, commenting or being mentioned on the commentable of this comment
   def involved_users
-    users = Set.new
-    users_mentioned = Set.new
+    users = []
```

Comment:
using set for users should be more efficient (the change in users_mentioned makes sense)

Reply:
I would guess the same - Sets are underused! :)